webservices/common/models.py

```diff
@@ -855,6 +856,17 @@ class Filings(db.Model):
     update_date = db.Column(db.Date)
 
     @property
+    def document_description(self):
+        if self.report_type_full and self.report_type_full != '':
+            clean_report = report_typere.sub(r'\{[^)]*\}', '', self.report_type_full)
```

Comment:
I don't see a variable called `reportype_re`. Should this be `re.sub(...)`? Also, like most regex logic, it would be helpful to include a docstring and unit test(s) here.